packages/browser/src/tracing/request.ts

```diff
@@ -113,10 +117,44 @@ export function instrumentOutgoingRequests(_options?: Partial<RequestInstrumenta
   const shouldAttachHeadersWithTargets = (url: string): boolean => shouldAttachHeaders(url, tracePropagationTargets);
 
   const spans: Record<string, Span> = {};
+  const client = getClient();
```

Comment:
instead of this, can we just pass the client in as an argument? At call-site we always have a client, so this should simplify this a bit!